apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -112,6 +113,12 @@ var TopInstructions = React.createClass({
     documentationUrl: PropTypes.string,
     ttsMarkdownInstructionsUrl:  PropTypes.string,
     levelVideos: PropTypes.array,
+    // TODO (epeach) - remove after resources tab A/B testing
+    // provides access to script id for level data
+    app: PropTypes.string,
```

Comment:
just curious whether this TODO applies only to the following line, or to all following lines in the block? No need to change it if you don't want to, since this comment is temporary.

Reply:
All lines following the block. The goal is that once the test is over, all of the changes in this PR can be reverted.